### PR TITLE
Use `env` wherever `prebuildopts is used to set environmental variables.

### PR DESCRIPTION
**Pull request by riccardomurri**
As requested by @boegel in PR #2807

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
@riccardomurri we have a single instance of this for `preinstallopts` too:

```
./easybuild/easyconfigs/p/p4vasp/p4vasp-0.3.29-intel-2016a-Python-2.7.11.eb:preinstallopts = "HOME=%(installdir)s make local &&"
```

let's fix that in here too?

**Comment by boegel** [comment]
@riccardomurri merging #2807 resulted in some conflicts here due to lines in context being modified, please resync with `develop`?

**Comment by riccardomurri** [comment]
(Kenneth Hoste, Thu, Mar 31, 2016 at 09:06:10AM -0700:)
> @riccardomurri merging #2807 resulted in some conflicts here due to
> lines in context being modified, please resync with `develop`?

Done, but I had to do a `push -f`.

File: easybuild/easyconfigs/m/Mesa/Mesa-10.4.5-intel-2015a-Python-2.7.9.eb [hunk 1]
```diff
@@ -59,7 +59,7 @@ configopts += " --disable-driglx-direct --with-gallium-drivers='' --disable-egl"
 # package-config files for os dependencies are in an os specific place
 #preconfigopts = ' PKG_CONFIG_PATH="$PKG_CONFIG_PATH:/usr/lib64/pkgconfig/:/usr/share/pkgconfig" '
 
-prebuildopts = 'CPATH="$EBROOTLIBDRM/include/libdrm" '
+prebuildopts = 'env CPATH="$EBROOTLIBDRM/include/libdrm" '
```
Comment (boegel) [inline easybuild/easyconfigs/m/Mesa/Mesa-10.4.5-intel-2015a-Python-2.7.9.eb]: eek, I don't think we should be hard redefining `$CPATH` and totally ignore the current value here...

@riccardomurri fixed with https://github.com/riccardomurri/easybuild-easyconfigs/pull/5

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 23 out of 24 (24 easyconfigs in this PR)
node2140.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7ec69d8d854c2bf2c64e8546253bef70 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 23 out of 24 (24 easyconfigs in this PR)
node2116.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/3d60a829b870e66b0b0c19a2de64adfb for a full test report.

**Comment by boegel** [comment]
The issue with motif 2.2.4 with `goolf/1.4.10` is not caused by the changes here, cfr. #2817.

Going in, thanks @riccardomurri!

**Pull request merged**